packages/xchain-ethereum/src/utils.ts

```diff
@@ -1,30 +1,32 @@
 import { Fees, Network as XChainNetwork } from '@xchainjs/xchain-client'
-import { baseAmount } from '@xchainjs/xchain-util/lib'
+import { baseAmount } from '@xchainjs/xchain-util'
 import { Network as EthNetwork } from './types'
 
 export const ETH_DECIMAL = 18
 
+/**
+ * XChainNetwork -> EthNetwork
+ *
+ * @param {XChainNetwork} network
+ * @returns {EthNetwork}
+ */
 export const xchainNetworkToEths = (network: XChainNetwork): EthNetwork => {
   switch (network) {
-    /**
-     * DO NOT use switch/case's default branch
-     * to be sure that ALL possible cases are
-     * processed in a similar way to reverted ethNetworkToXchains
-     */
```

Comment:
Let's keep this helpful comment, but comment it using `// ...` if we do need to avoid it to get generated as documentation.